config/schedule.rb

```diff
@@ -67,3 +67,8 @@
 every '0 6 1 * *', roles: [:prod] do
   rake " lib_jobs:send_eads"
 end
+
+# Run on production every Thursday at 4am
```

Comment:
Usually the times in the task itself are UTC, then we translate to EST/EDT in the comment - double check if it's supposed to run 4 am UTC or EDT, and update comment or time accordingly. Also note if this is dependent on any other lib-job or task from Alma.